services/wallet/api.go

```diff
@@ -341,6 +341,13 @@ func (api *API) GetCollectiblesByUniqueIDAsync(requestID int32, uniqueIDs []thir
 	return nil
 }
 
+func (api *API) FetchCollectibleSocialsAsync(requestID int32, uniqueID thirdparty.CollectibleUniqueID) error {
```

Comment:
hmm if Socials are tied to a collection and not a specific collectible, we should make the API collection-based